code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -10,6 +10,10 @@
 		act = copytext(act, 1, t1)
 
 	var/muzzled = is_muzzled()
+	if(muzzled)
+		var/obj/item/clothing/mask/muzzle/M = wear_mask
+		if(!M.mute)
```

Comment:
Should `M.mute == 1` prevent the emote from functioning here? Since a `mute` of `2` is required to inhibit speech.
Also, it would be wise to change `1` and `2` in this context into #define flags, perhaps stored in `code/__DEFINES/mob.dm`